experimental/cache_line_analysis.cc

```diff
@@ -40,21 +40,20 @@ double_t FindReadingTime(std::vector<char> buf, int64_t sz, int step) {
 // the cache line size, so not all cache lines are needed to be read from memory
 // to cache
 // Note that (1) the time of reading from cache is negiligble compared to
-// reading from memory, and (2) CPUs read cache lines from memory instead byte
-// by byte.
+// reading from memory, and (2) CPUs read memory in blocks of entire cache
+// lines, rather than reading individual bytes.
 // Therefore, the time for reading a fixed size buffer (smaller than cache size)
 // in chunks of size n should be around the same value as long as n < cache line
 // size (the average reading time increases as n increase (i.e. total time/
 // number of reads)), and then as n increases less number of cache lines are
 // read from memory and takes less time.
-
-int cache_line_size_analysis() {
+int main() {
   static constexpr int64_t kMaxSize = 256;
   static constexpr int64_t kMinSize = 4;
-  static constexpr int iterations = 100;
-
-  int64_t sz = 4 * 1024 * 1024;
-  int64_t cache_size = 8 * 1024 * 1024;
+  static constexpr int kIterations = 10;
+  static constexpr int64_t cache_size = 8 * 1024 * 1024;
+  
+  static constexpr int64_t sz = 4 * 1024 * 1024;
```

Comment:
Ideally avoid abbreviations like "`sz`" that drop letters between words -- see [style guide entry](https://google.github.io/styleguide/cppguide.html#General_Naming_Rules)

Should probably be kSize. (Maybe an even longer name to distinguish between "size" and "cache size".)